framework/docs/src/3_framework/0_prerequisites.md

```diff
@@ -75,7 +75,19 @@ The application of the actor model in the CosmWasm framework provides the follow
   prevents
   inconsistent states in the contract.
 
-To learn more about CosmWasm, check out the <a href="https://book.cosmwasm.com/" target="_blank">official
+#### Coming from EVM?
+
+There are a few key differences between the EVM and CosmWasm that you should be aware of. The most important one is that instances of contracts and the code that they run against are two different concepts in CosmWasm. This means that you can have multiple instances of the same contract code running at the same time, each with their own state. This is not possible in EVM, where the contract code and the contract instance are the same thing.
```

Comment:
We can mention that this leads to significantly reduced gas cost in instantiating new contracts and in complex contract interactions.

Reply:
I'd rather keep this section factual but will add a link to comparison article if they want to deep-dive the differences and advantages!